src/ngraph/runtime/cpu/pass/cpu_layout.cpp

```diff
@@ -256,10 +261,16 @@ static void set_native_layouts(runtime::cpu::CPU_ExternalFunction* external_func
         auto shape = tv->get_shape();
         auto et = tv->get_element_type();
         auto layout = std::make_shared<ngraph::runtime::cpu::LayoutDescriptor>(*tv);
-        if (mkldnn_utils::can_create_mkldnn_md(shape, layout->get_strides(), et))
+        auto strides = layout->get_strides();
+
+        if (shape == Shape{})
+        {
+            shape = Shape{1};
+            strides = Strides{1};
+        }
+        if (mkldnn_utils::can_create_mkldnn_md(shape, strides, et))
         {
-            auto native_md =
-                mkldnn_utils::create_blocked_mkldnn_md(shape, layout->get_strides(), et);
+            auto native_md = mkldnn_utils::create_blocked_mkldnn_md(shape, strides, et);
```

Comment:
would be cleaner/less code duplication to modify create_blocked_mkldnn_md instead of creating Shape{1} here.